### PR TITLE
hardenings for various profiles

### PR DESCRIPTION
**Pull request by glitsj16**
Based on daily used .local files on Arch and Ubuntu. I would appreciate a review before merging. TIA.

### CODE REVIEW

File: etc/claws-mail.profile [hunk 1]
```diff
@@ -15,9 +15,11 @@ noblacklist ${HOME}/Mail
 
 include disable-common.inc
 include disable-devel.inc
+include disable-exec.inc
 include disable-interpreters.inc
 include disable-passwdmgr.inc
 include disable-programs.inc
+include disable-xdg.inc
```
Comment (rusty-snake) [inline etc/claws-mail.profile]: Maybe `noblacklist ${DOCUMENTS}`.

File: etc/curl.profile [hunk 2]
```diff
@@ -34,6 +38,7 @@ protocol inet,inet6
 seccomp
 shell none
 tracelog
+x11 none
```
Comment (rusty-snake) [inline etc/curl.profile]: `x11 none` without `net none` (which makes no sense for curl) breaks on some systems.

```suggestion

```

File: etc/devilspie.profile [hunk 3]
```diff
@@ -8,6 +8,8 @@ include globals.local
 
 noblacklist ${HOME}/.devilspie
 
+blacklist /tmp/.X11-unix
```
Comment (rusty-snake) [inline etc/devilspie.profile]: implied by `x11 none`.

File: etc/dig.profile [hunk 4]
```diff
@@ -40,6 +42,8 @@ novideo
 protocol unix,inet,inet6
 seccomp
 shell none
+tracelog
+x11 none
```
Comment (rusty-snake) [inline etc/dig.profile]: see curl

File: etc/dnscrypt-proxy.profile [hunk 5]
```diff
@@ -34,6 +39,9 @@ nou2f
 novideo
 protocol inet,inet6
 seccomp.drop _sysctl,acct,add_key,adjtimex,clock_adjtime,delete_module,fanotify_init,finit_module,get_mempolicy,init_module,io_cancel,io_destroy,io_getevents,io_setup,io_submit,ioperm,iopl,kcmp,kexec_file_load,kexec_load,keyctl,lookup_dcookie,mbind,migrate_pages,modify_ldt,mount,move_pages,open_by_handle_at,perf_event_open,perf_event_open,pivot_root,process_vm_readv,process_vm_writev,ptrace,remap_file_pages,request_key,set_mempolicy,swapoff,swapon,sysfs,syslog,umount2,uselib,vmsplice
+shell none
+tracelog
+x11 none
```
Comment (rusty-snake) [inline etc/dnscrypt-proxy.profile]: see curl

File: etc/easystroke.profile [hunk 6]
```diff
@@ -16,10 +16,15 @@ include disable-passwdmgr.inc
...
 
+mkdir ${HOME}/.easystroke
+whitelist ${HOME}/.easystroke
+include whitelist-common.inc
 include whitelist-usr-share-common.inc
+include whitelist-var-common.inc
 
 apparmor
 caps.drop all
+ipc-namespace
```
Comment (rusty-snake) [inline etc/easystroke.profile]: Unsure

File: etc/wget.profile [hunk 7]
```diff
@@ -43,6 +43,7 @@ protocol unix,inet,inet6
 seccomp
 shell none
 tracelog
+x11 none
```
Comment (rusty-snake) [inline etc/wget.profile]: see curl

File: etc/claws-mail.profile (hunk 1 above)
Comment (glitsj16) [reply etc/claws-mail.profile]: Done, nice suggestion.

File: etc/devilspie.profile (hunk 3 above)
Comment (glitsj16) [reply etc/devilspie.profile]: Done.

**Comment by glitsj16** [comment]
@rusty-snake I made the necessary changes. Thanks for the review, very much appreciated! I really should make a **big** note on the `x11 none` mishaps...

File: etc/devilspie.profile (hunk 3 above)
Comment (rusty-snake) [reply etc/devilspie.profile]: :mag: :cold_sweat:

**Pull request merged**